src/centralserver/admin-ui/src/views/AppBase.vue

```diff
@@ -0,0 +1,113 @@
+   in the Software without restriction, including without limitation the rights
+   to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+   copies of the Software, and to permit persons to whom the Software is
+   furnished to do so, subject to the following conditions:
+
+   The above copyright notice and this permission notice shall be included in
+   all copies or substantial portions of the Software.
+
+   THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+   IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+   FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+   AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+   LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+   OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+   THE SOFTWARE.
+ -->
+
+<template>
+  <div>
+    <router-view name="top" />
+    <v-layout align-center justify-center>
+      <transition name="fade" mode="out-in">
+        <div class="base-full-width">
+          <router-view name="subTabs" />
+          <router-view name="alerts" />
+          <v-layout
+            align-center
+            justify-center
+            class="base-full-width bottom-pad"
+          >
+            <router-view />
+          </v-layout>
+        </div>
+      </transition>
+    </v-layout>
+
+    <v-dialog v-if="showDialog" v-model="showDialog" width="500" persistent>
+      <v-card class="xrd-card">
+        <v-card-title>
+          <span class="headline">{{ $t('logout.sessionExpired') }}</span>
+        </v-card-title>
+        <v-card-text class="pt-4">{{ $t('logout.idleWarning') }}</v-card-text>
+        <v-card-actions class="xrd-card-actions">
+          <v-spacer></v-spacer>
+          <xrd-button @click="logout()">{{ $t('action.ok') }}</xrd-button>
+        </v-card-actions>
+      </v-card>
+    </v-dialog>
+  </div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import { RouteName } from '@/global';
+
+export default Vue.extend({
+  data() {
+    return {
+      sessionPollInterval: 0,
+      alertsPollInterval: 0,
+    };
+  },
+  computed: {
+    showDialog(): boolean {
+      return this.$store.getters.isSessionAlive === false;
+    },
+  },
+  created() {
+    // Set interval to poll backend for session
+  },
+  methods: {
+    pollSessionStatus() {
+      // Do polling
+    },
+    logout(): void {
+      this.$store.dispatch('logout');
+      this.$router.replace({ name: RouteName.Login });
+    },
+  },
+});
+</script>
+
+<style lang="scss" scoped>
+@import '@/assets/shared';
+.base-full-width {
+  width: 100%;
+  padding-bottom: 40px;
+}
+
+.bottom-pad {
+  padding-bottom: 40px;
+}
+
+.fade-enter-active,
+.fade-leave-active {
+  transition-duration: 0.2s;
+  transition-property: opacity;
+  transition-timing-function: ease;
+}
+.fade-enter,
+.fade-leave-active {
+  opacity: 0;
+}
```

Comment:
These styles aren't directly used. Are they customisation for vuetify components or something like that or could they be removed?

Reply:
They are used by transition on line 31.  <transition name="fade" mode="out-in">
Vue picks up the classes by the "fade" name.
https://v1.vuejs.org/guide/transitions.html#Example